.github/workflows/ci-run-test.yml

```diff
@@ -338,7 +338,11 @@ jobs:
         if: ${{ inputs.platform != 'windows-latest' }}
         run: |
           cd sdk/nodejs
-          if [ -e .nyc_output ]; then yarn run nyc report -r cobertura --report-dir "$PULUMI_TEST_COVERAGE_PATH"; fi
+          if [ -e .nyc_output ]; then
+            TS=$(python -c 'import time; print(int(time.time() * 1000))')
```

Comment:
Would we be better of with a uuid here?  It's probably unlikely that this is happening at the same time, but some of these tests are run in parallel, so they might end up getting the same timestamp, and we end up with the same clash.

Reply:
Yes, potentially. I copied this from Go tests, but maybe I can change them all.

Reply:
Done now